CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### PaymentsUI
 * [Added] Adds support for [card brand choice](https://stripe.com/docs/card-brand-choice) to STPPaymentCardTextField and STPCardFormView. To set a default preferred network for these UI elements, use the new `preferredNetworks` parameter.
 
+* [Deprecated] Mark STPPinManagementService deprecated & suggest alternative.
```

Comment:
1. Will this require a major version change, or a minor version is acceptable?
2. Do we need to update the MIGRATING.md file? 
3. Seems like we've avoided using [Deprecated], and use [Changed] instead. ([example](https://github.com/stripe/stripe-ios/blob/master/CHANGELOG.md?plain=1#L372))

Reply:
done